app/controllers/public/index.js

```diff
@@ -135,11 +136,10 @@ export default class IndexController extends Controller {
         })
         .catch(async e => {
           console.error('Error while saving order', e);
-          try {
-            await Promise.allSettled((attendees ? attendees.toArray() : []).map(attendee => attendee.destroyRecord()));
-          } catch (error) {
-            console.error('Error while deleting attendees after order failure', error);
-          }
+          await RSVP.allSettled(attendees ? attendees.toArray() : []).then(attendee => attendee.destroyRecord(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['RSVP' is not defined.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5583663)